src/executors/source-replacement/index.test.ts

```diff
@@ -0,0 +1,19 @@
+describe('source-replacement', () => {
```

Comment:
```suggestion
describe('source-replacement()', () => {
```

Reply:
This executor does not export as function